extensions/ql-vscode/src/remote-queries/view/LastUpdated.tsx

```diff
@@ -23,7 +25,7 @@ const LastUpdated = ({ lastUpdated }: Props) => (
         <RepoPushIcon size={16} />
       </IconContainer>
       <Duration>
-        {humanizeDuration(lastUpdated)}
+        {humanizeDuration(lastUpdated && -lastUpdated)}
```

Comment:
I'm not sure I understand this line of code — could you explain it?

Reply:
I was about to ask the same question! What does it mean to `&&` these two numbers? 😅

Reply:
lastUpdated may be undefined. If so, do not want to negate it (this would convert it to NaN, which is BAD).

If lastUpdated is 0, then negating is a noop. So we don't need to negate.

Would it be easier to understand if I did this?

```

lastUpdated === undefined ? undefined : -lastUpdated
```

Reply:
```suggestion
        {humanizeDuration(lastUpdated === undefined ? undefined : -lastUpdated)}
```

Reply:
Hm, yes, I find that easier to understand. What happens if `lastUpdated` is an integer other than 0 though? Do we expect it to be a positive or negative integer (and why do we need to negate it at all if the absolute value is taken in `humanizeDuration()`?)

Reply:
Good questions. If `lastUpdated` is something other than 0, then it is negated (meaning, a time in the past).

Absolute value is used to determine which unit to use. A positive integer indicates "XXX units _from now_". A negative integer indicates "XXX units ago".

Maybe it would be clearer if we ensured that `lastUpdated` was always negative and I can put a comment there.

Reply:
oh yes! I see how it works with the new change now. I think that would make it clearer, yes!